src/Resources/scaffolds/6.0/change-password/src/Form/User/ChangePasswordFormType.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace App\Form\User;
```

Comment:
That's a deep directory/namespace. `ChangePasswordFormType` and `RegistrationFormType` are both pretty obvious to me that they relate to users. So I'd prefer not to have the extra directory. Same with `Controller/User`.